src/components/NibrsStackedBar.js

```diff
@@ -48,8 +48,7 @@ class NibrsStackedBar extends React.Component {
 
     return (
       <div className="mb2 pb2 border-bottom border-blue-light">
-        <div className="mb1 blue bold">{title}</div>
-        <div className="mb1 fs-12 bold caps red">Incidents</div>
+        <div className="mb2 blue bold">{title}</div>
```

Comment:
I'm fine with merging this now but should we just use the proper unit? So it would say "Victims" or "Offender"?